bin/run_genericcamera.py

```diff
@@ -1,6 +1,28 @@
 #!/usr/bin/env python
+#
+# This file is part of ts_GenericCamera.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the Vera Rubin Observatory
+# Project (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
 import asyncio
 
 from lsst.ts.GenericCamera import GenericCameraCsc
 
-asyncio.run(GenericCameraCsc.amain(index=None))
+asyncio.run(GenericCameraCsc.amain(index=10))
```

Comment:
`index=10`?


I think this needs to be `index=True` so salobj will handle it. Bottom line, salobj should add a position argument so one would run the CSC as:

```
run_genericcamera.py 1
```

Reply:
Thanks for catching that. That's a left over from my attempts to get DDS to work on the raspberry pi. It turned out that the firewall was blocking all traffic and I forgot to revert this. Done.